lib/src/foundation/feature_discovery.dart

```diff
@@ -29,20 +29,20 @@ class FeatureDiscovery extends StatelessWidget {
   /// Though they can be placed in any [Iterable], it is recommended to pass them as a [Set]
   /// because this ensures that every step is only shown once.
   static void discoverFeatures(BuildContext context, Iterable<String> steps) =>
-      _blocOf(context).discoverFeatures(steps: steps.toList());
+      _blocOf(context).discoverFeatures(steps.toList());
 
   /// This will schedule completion of the current discovery step and continue
   /// onto the step after the completion animation of the current overlay if successful.
   static void completeCurrentStep(BuildContext context) =>
       _blocOf(context).completeStep();
 
-  /// This will schedule dismissal of the current discovery step and with that
-  /// of the current feature discovery. The dismissal animation will play if successful.
+  /// This will dismiss all steps.
+  /// `onDimiss` parameter will be ignored for every active overlay.
```

Comment:
@creativecreatorormaybenot I also edited this documentation. Tell me what you think about it.

Reply:
I will add an article.